verification/curator-service/test/home.test.ts

```diff
@@ -1,6 +1,21 @@
 import request from 'supertest';
 import app from '../src/index';
 
+import mongoose from 'mongoose';
+
+beforeAll(() => {
+    return mongoose.connect(
```

Comment:
Is this necessary, in this file?

My current understanding is that MongoDB can live with a bum connection to localhost until a request is made. As with the other data-service tests, if we set up dotenv with a reasonable `devDefault` for the DB connection URI, we won't need to do this.

I was recently looking into the way jest-mongodb works, since it appears somewhat nuanced. When a test with jest-mongodb is run, it seems the primary connection switches from whatever's specified by the default app config (e.g. a localhost address) to the custom address used for the jest instance. This to say that if the test doesn't actually use MongoDB, it shouldn't matter.

Reply:
OK I wasn't really keen on letting mongoose hang on a bum connection during tests, I seemed to notice slightly speedier tests when I added that but let's go for less code, less is better :)